README.md

```diff
@@ -5,7 +5,8 @@ Samples that demonstrate how to use various Prism features with Windows 10 UWP.
 -----------|-------------|
 | [AdventureWorks.Shopper][1] | Complete AdventureWorks sample, showing more advanced application.
 | [ExtendedSplashScreenDemo][2] | How to use show a splashscreen on starting your app.
-
+| [SplitViewSample][3] | Split View Prism Example without Unity (Using PrismApplication)
```

Comment:
public sealed partial class App : PrismUnityApplication
I think it's using Unity :)